<?xml version="1.0" encoding="UTF-8"?><testrun name="BillServiceImplTest" project="bill" tests="4" started="4" failures="1" errors="0" ignored="0">
  <testsuite name="org.medi.ocean.service.BillServiceImplTest" time="0.009">
    <testcase name="testGenerateBillAndTaxBasedOnCategoryAAndB" classname="org.medi.ocean.service.BillServiceImplTest" time="0.007">
      <failure>java.lang.AssertionError: expected:&lt;0&gt; but was:&lt;105&gt;&#13;
	at org.junit.Assert.fail(Assert.java:88)&#13;
	at org.junit.Assert.failNotEquals(Assert.java:834)&#13;
	at org.junit.Assert.assertEquals(Assert.java:118)&#13;
	at org.junit.Assert.assertEquals(Assert.java:144)&#13;
	at org.medi.ocean.service.BillServiceImplTest.testGenerateBillAndTaxBasedOnCategoryAAndB(BillServiceImplTest.java:74)&#13;
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)&#13;
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)&#13;
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)&#13;
	at java.lang.reflect.Method.invoke(Unknown Source)&#13;
	at junit.framework.TestCase.runTest(TestCase.java:176)&#13;
	at junit.framework.TestCase.runBare(TestCase.java:141)&#13;
	at junit.framework.TestResult$1.protect(TestResult.java:122)&#13;
	at junit.framework.TestResult.runProtected(TestResult.java:142)&#13;
	at junit.framework.TestResult.run(TestResult.java:125)&#13;
	at junit.framework.TestCase.run(TestCase.java:129)&#13;
	at junit.framework.TestSuite.runTest(TestSuite.java:252)&#13;
	at junit.framework.TestSuite.run(TestSuite.java:247)&#13;
	at org.junit.internal.runners.JUnit38ClassRunner.run(JUnit38ClassRunner.java:86)&#13;
	at org.eclipse.jdt.internal.junit4.runner.JUnit4TestReference.run(JUnit4TestReference.java:86)&#13;
	at org.eclipse.jdt.internal.junit.runner.TestExecution.run(TestExecution.java:38)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:459)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:675)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.run(RemoteTestRunner.java:382)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.main(RemoteTestRunner.java:192)&#13;
&#13;
</failure>
    </testcase>
    <testcase name="testGenerateBillAndTaxBasedOnCategoryA" classname="org.medi.ocean.service.BillServiceImplTest" time="0.0"/>
    <testcase name="testGenerateBillAndTaxBasedOnCategoryB" classname="org.medi.ocean.service.BillServiceImplTest" time="0.0"/>
    <testcase name="testGenerateBillAndTaxBasedOnCategoryC" classname="org.medi.ocean.service.BillServiceImplTest" time="0.001"/>
  </testsuite>
</testrun>
